### ISSUE TITLE
Website request: filled preview

### ISSUE THREAD

**Issue by ZAYDEK** [feathericons/feather#678]
Hi Cole. I love the project.

I would love if there was a streamlined way to preview the icons using the fill technique mentioned in #225. If the Figma plugin is a something you care deeply about, you might consider adding a technique to adding it there, too.

As far as the website design goes, one recommendation is that you could possibly have an iOS-like toggle button to the right of the search bar for enabling/disabling and perhaps a tooltip next to it that briefly explains how to achieve the effect with CSS.

Anyway, thanks!

**Comment by locness3** [comment]
Well, the spirit of feather icons is to have no fills. I guess your mind can do the prview itself. :)

**Comment by ZAYDEK** [comment]
I just did a test and found that the icons aren’t exactly interoperable as filled.

<img width="1440" alt="Screen Shot 2019-07-13 at 8 19 54 PM" src="https://user-images.githubusercontent.com/6156990/61170967-cc794100-a5ab-11e9-88c7-e2cf02a332ff.png">

**Comment by afuno** [comment]
@zaydek I agree with you. Many icons are not suitable for using `fill` or `stroke` coloring.
